sys-devel/clang-common/clang-common-16.0.6-r2.ebuild

```
@@ -13,7 +13,7 @@ SLOT="0"
 KEYWORDS="amd64 arm arm64 ~loong ppc ppc64 ~riscv sparc x86 ~amd64-linux ~ppc-macos ~x64-macos"
 IUSE="
 	default-compiler-rt default-libcxx default-lld llvm-libunwind
-	hardened
+	hardened bootstrap-prefix
```

Let's keep things sorted, and perhaps keep `default*` group separate (i.e. move `llvm-libunwind` from the line above):

```suggestion
	bootstrap-prefix hardened llvm-libunwind
```

---

done